andela/mrm-mobile
# #159275744 Update the App Launcher Icon and name

#### What does this PR do?

Update the App Launcher Icon to match what is provided in the Invision Mockup.

#### How should this be manually tested?

* Run the application on a physical device
* Navigate to the home view of the device to see the change.

#### Any background context you want to provide?

* The app was showing the android Launcher Icon.
* The name for the application was `MRM`

#### What are the relevant pivotal tracker stories?

[#159275744](https://www.pivotaltracker.com/story/show/159275744)

#### Screenshots (if appropriate)

![screenshot_2018-07-31-19-41-17](https://user-images.githubusercontent.com/15923109/43474117-a5e6465e-94fa-11e8-9b92-ed23f34db189.png)
